skyportal/handlers/api/source.py

```diff
@@ -289,10 +290,22 @@ async def get_source(
     duplicates = session.scalars(
         Source.select(user).join(duplicate_objs, Source.obj_id == duplicate_objs.c.id)
     ).all()
-    if len(duplicates) > 0:
-        source_info["duplicates"] = list({dup.obj_id for dup in duplicates})
-    else:
-        source_info["duplicates"] = None
+    # ironically, we might have duplicates of duplicates because we can have multiple sources for the same obj_id
+    # so, deduplicate to only keep one source per obj_id, for that we take advantage of dictionaries and their unique keys
+    # TODO: replace this query by something better where we have only one entry per obj_id
```

Comment:
Anything to do with the TODO here?

Reply:
Not really the query is fine as is, not worth spending more time on it (unless prod goes bananas). Just updated the comment to look not so crazy